jujugui/static/gui/src/app/components/user-profile/entity/entity.js

```diff
@@ -358,6 +468,9 @@ YUI.add('user-profile-entity', function() {
         });
         destroyButton = this._generateDestroyButton(entity);
       } else {
+        if (isCharm) {
+          this._getMetrics();
```

Comment:
Because the app re-renders on every db update this will cause an xhr request every 5s or so when it's open so I'm not sure it's a good idea to defer.